src/renderer/views/Watch/Watch.js

```diff
@@ -765,6 +765,14 @@ export default Vue.extend({
       })
     },
 
+    getWatchedProgress: function () {
+      return this.$refs.videoPlayer && this.$refs.videoPlayer.player ? this.$refs.videoPlayer.player.currentTime() : 0
```

Comment:
@PrestonN I'm concerned by how often this gets called, given that currentTime() can update on sub-second values, but my attempts to debounce this resulted in the value always coming back as undefined, so I'm not sure how to optimize further.

Reply:
Not too sure why there would be issues with this getting called to often.  If the solutions I've mentioned here doesn't solve this problem then please elaborate on why you think this would get repeatedly called.

Reply:
That concern will no longer exist if I am able to call getTimestamp() from a child component at the time the user attempts to copy/open, rather than that value needing to be available as a prop.